syntax = "proto3";

package rpcheader;

message RequreMsg
{
    bytes service_name=1;
    bytes method_name=2;
    int32 args_size=3;
}
